src/main/java/com/google/alpollo/YouTubeService.java

```diff
@@ -0,0 +1,33 @@
+package com.google.alpollo;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.YouTubeRequestInitializer;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
+public class YouTubeService {
+  private static final JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
```

Comment:
Is there a reason to use Jackson here and not Gson like in other parts of the project?